src/writing-your-own-application/async.md

```diff
@@ -0,0 +1,29 @@
+# Async
+
+This subchapter does not serve as an `async` tutorial or teaching material. For these purposes visit [official async-book]. 
+
```

Comment:
We should at least say something like `embassy` is supported by esp-hal ... just starting to talk about embassy looks a bit odd otherwise

While the next paragraph explains it, just starting to explain what `embassy` is feels weird

Reply:
I re-order the paragraph a bit, does it look better or does it need improvement?